src/app/services/actions/adaptive-cards-action-creator.ts

```diff
@@ -50,21 +54,25 @@ export function getAdaptiveCard(payload: string, sampleQuery: IQuery): Function
 
     dispatch(getAdaptiveCardPending());
 
-    return fetch(`https://templates.adaptivecards.io/graph.microsoft.com/${templateFileName}`)
-      .then(resp => resp.json())
+    return fetch(
+      `https://templates.adaptivecards.io/graph.microsoft.com/${templateFileName}`
```

Comment:
[Nit] the indenting is off here

Reply:
Which formatting tool is recommended for use?

Reply:
@ElinorW #741 adds some vscode settings that will help with the formatting.